### PR TITLE
wip

### PR DESCRIPTION
**Pull request by farmisen**
Embedable widget - first iteration

### CODE REVIEW

File: app/views/widget/capture_payment.html.slim [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    hr
+  .payments
+    .tabs.is-boxed.is-fullwidth
+      ul
+        li.card
+          a data-type='card'
+            | Card
+        li.dwalla
+          a data-type='dwalla'
+            | Dwalla
```
Comment (elliottjf) [inline app/views/widget/capture_payment.html.slim:13]: misspelling here, it's 'Dwolla'

File: app/assets/javascripts/jschannel.js [hunk 2]
```diff
@@ -0,0 +1,620 @@
+/*
```
Comment (elliottjf) [inline app/assets/javascripts/jschannel.js:1]: looks like a duplicate of public/jschannel.js.  which is used?

File: app/views/widget/capture_payment.html.slim (hunk 1 above)
Comment (farmisen) [inline app/views/widget/capture_payment.html.slim:13]: I thought I fixed all of them. Good catch.

File: app/assets/javascripts/jschannel.js (hunk 2 above)
Comment (farmisen) [inline app/assets/javascripts/jschannel.js:1]: Actually the jschannel library is used in two different contexts. It's first injected in the page where the widget is embedded and I use this file to only include what's necessary but it is also used in the widget iframe itself and in this case it is coming from the js bundle  packaged by the rails asset pipeline. Few things I'd like to change, 1) bundle and minify all the js injected in the host page 2) only use the necessary js and css assets for the widget itself excluding everything that is/will be used in the web site.

**Pull request merged**